README.md

```diff
@@ -2,62 +2,59 @@
 
 ## Description ##
 
-*Protean* is a library for representing hierarchical data in a uniform way
-across multiple platforms.
+`protean` is a library for representing hierarchical data in a uniform way across multiple platforms.
 
-The library defines *a variant object* that can hold primitive data as well as
-collections (e.g. lists, dictionaries) with arbitrary depth. The variant can be
-inspected, enumerated, and/or serialised to XML/binary format.
+The library defines a `variant` object that can hold primitive data as well as collections (e.g. lists, dictionaries) with arbitrary depth. The variant can be inspected, enumerated, and/or serialised to XML/binary format.
 
-Protean has implementations in `C++` and `C#`.
+Protean has implementations in C++, C# and Python.
 
 ## Compilation ##
```

Comment:
This section appears C++-specific and make no mention of building the C# version.

Reply:
This is true, but it was equally true before; I was just updating what was already there. I will leave the addition of C# instructions to the expert around here!

Reply:
Would you mind putting a C++ sub-header in then? I'll add the C#-specific stuff another time.

Reply:
Sure; done.